virttest/utils_test/libvirt.py

```diff
@@ -3575,7 +3576,11 @@ def customize_libvirt_config(params,
     :param config_object: an existing utils_config.LibvirtConfigCommon object
     :return: utils_config.LibvirtConfigCommon object
     """
-
+    # Hardcode config_type to virtqemud under modularity daemon mode when config_type="libvirtd"
+    # Otherwise accept config_type as it is
+    if utils_split_daemons.is_modular_daemon():
+        if config_type in ["libvirtd"]:
+            config_type = "virtqemud"
```

Comment:
The function "customize_libvirt_config" was used to modifying config keys for 'libvirtd', 'qemu', 'sysconfig' and so on, would you please create a new function to get the config_type?

The configuration key of libvirtd does not map directly to those in virtqemud. And some virt*d daemons have the same configuration key, but for different functions. So I'm not sure if it works... @yafu-1  Could you please help review it?

Reply:
if carefully check the difference between this PR and https://github.com/autotest/tp-libvirt/pull/3023/files or https://github.com/autotest/tp-libvirt/pull/3012, the only difference is moved below logic from outside of customize_libvirt_config to inside of customize_libvirt_config :
config_type = 'libvirtd'
if utils_split_daemons.is_modular_daemon():
            config_type = "virtqemud"

Reply:
I think the pr works well. The default config file is 'virtqemud.conf' if not specified and can set to different split daemon conf file if needed.